WebHostLib/static/assets/tutorial/archipelago/setup_en.md

```diff
@@ -1,79 +1,85 @@
+under any game. Clicking "Export Settings" in a game's settings page will download the YAML to your system. Games
+page: [Archipelago Games List](/games)
+
 In a multiworld there must be one YAML per world. Any number of players can play on each world using either the game's
-native coop system or using archipelago's coop support. Each world will hold one slot in the multiworld and will have a
-slot name and, if the relevant game requires it, files to associate it with that multiworld. If multiple people plan to
-play in one world cooperatively then they will only need one YAML for their coop world, but if each player is planning on
-playing their own game then they will each need a YAML. These YAML files can be generated by going to the relevant game's
-player settings page, entering the name they want to use for the game, setting the options to what they would like to
-play with and then clicking on the export settings button. This will then download a YAML file that will contain all of
-these options and this can then be given to whoever is going to generate the game.
-
-### Gather all player YAMLS
-All players that wish to play in the generated multiworld must have a YAML file which contains the settings that they wish to play with.
-A YAML is a file which contains human readable markup. In other words, this is a settings file kind of like an INI file or a TOML file. 
-Each player can go to the game's player settings page in order to determine the settings how they want them and then download a YAML file containing these settings.
-After getting the YAML files of each participant for your multiworld game, these can all either be placed together in the 
-`Archipelago\Players` folder or compressed into a ZIP folder to then be uploaded to the [website generator](/generate).
-If rolling locally ensure that the folder is clear of any files you do not wish to include in the game such as the 
-included default player settings files.
+native coop system or using Archipelago's coop support. Each world will hold one slot in the multiworld and will have a
+slot name and, if the relevant game requires it, files to associate it with that multiworld.
+
+If multiple people plan to play in one world cooperatively then they will only need one YAML for their coop world. If
+each player is planning on playing their own game then they will each need a YAML.
+
+### Gather All Player YAMLs
+
+All players that wish to play in the generated multiworld must have a YAML file which contains the settings that they
+wish to play with.
+
+Typically, a single participant of the multiworld will gather the YAML files from all other players. After getting the
+YAML files of each participant for your multiworld game they can be compressed into a ZIP folder to then be uploaded to
+the multiworld generator page. Multiworld generator
+page: [Archipelago Seed Generator Page](https://archipelago.gg/generate)
+
+#### Rolling a YAML Locally
```

Comment:
The flow here is a bit confusing. I would recommend generating and hosting on the website be the core flow with an alternative path for generating and hosting locally. While many ppl who are active/been around generate locally, the vast majority of new players will be using the website

Reply:
This article is specifically for setup and usage of locally installed Archipelago server. There is another article for using the website.